recipes/default.rb

```diff
@@ -27,6 +27,42 @@
     not_if { node['sendmail_ses']['domain'] }
   end
 
+	if node['sendmail_ses']['secure_tunnel'] 
+		package 'stunnel'
+		frequency = 86400 * node['sendmail_ses']['cert_frequency']
+		smtp_host = 'localhost'
+		smtp_port = node['sendmail_ses']['secure_port'] || 2525
+
+		unless File.exist?("#{ node['sendmail_ses']['cert_file'] }") && File.mtime("#{ node['sendmail_ses']['cert_file'] }") > Time.now - frequency 
```

Comment:
add a not_if argument to the execute 'create cert' resource and move the unless statement there